packages/app/src/app/pages/VercelAuth/index.js

```diff
@@ -14,8 +14,9 @@ const VercelSignIn = () => {
 
   useEffect(() => {
     if (document.location.search.match(/\?code=(.*)/)) {
```

Comment:
I don't think this will work if the query params start with something else other than `?code=`

Eg. `?auth=true&code=123`

Reply:
Maybe `includes("code=")` would be a better option.

Reply:
moved code computation above the if